src/importexport/musicxml/internal/musicxml/importmxmlpass2.cpp

```diff
@@ -3002,7 +3013,13 @@ void MusicXMLParserDirection::direction(const String& partId,
             m_logger->logError(u"spanner already stopped", &m_e);
             delete desc.sp;
         } else {
-            if (spdesc.isStarted) {
+            if (spdesc.sp && spdesc.sp->isOttava() && delayOttava) {
```

Comment:
Is it correct that you don't need to check `spdesc.isStarted` here?

Reply:
It is the else branch of `if (spdesc._isStopped)`, can it have a 3rd state?

Reply:
@cbjeukendrup I probably ought to, it can't hurt to be safe